jina/hubble/helper.py

```diff
@@ -0,0 +1,59 @@
+"""Module for helper functions for Hub API."""
+
+import io
+from pathlib import Path
+
+
+def archive_package(package_folder: 'Path') -> 'io.BytesIO':
+    """
+    Archives the given folder in zip format and return a data stream.
+    :param package_folder: the folder path of the package
+    :return: the data stream of zip content
+    """
+    import os
+    import zipfile
+
+    # import pathspec
+
+    # TODO: get ignored file pattern from `resources/ignored_files`
+    ignored_file_specs = ['**/*.pyc']
+    ignored_dir_specs = [
+        '__MACOSX',
+        '.DS_Store',
+        '__pycache__',
+        '.eggs',
+        '*.egg-info',
+        '.git',
+        '.vscode',
+    ]
```

Comment:
how about:
- put this file https://github.com/github/gitignore/blob/master/Python.gitignore under `jina/resources`
- load it via
```python
from jina import __resources_path__
with open(os.path.join(__resources_path__, 'Python.gitignore')) as fp:
   ignored_dir_specs = fp.readlines()
```